woocommerce/woocommerce-android
# [Woo POS][Cash & Receipts] Prompt alert when woo core versions are no correct and receipt clicked

Closes: #13047


### Description

The PR adds toast message when a user clicks on "receipt" button and when WCCore version is not supported
### Testing information

Two scenarios:
* With WC >=9.5.0
* More -> POS -> Make purchase
* Click on "receipts"
* Notice that the mock screen is open

---

* With WC < 9.5.0
* More -> POS -> Make purchase
* Click on "receipts"
* Notice that the toast is shown

### The tests that have been performed

Above
### Images/gif


https://github.com/user-attachments/assets/bf052d4e-4fba-4365-8967-22ac510ea2d3


https://github.com/user-attachments/assets/cc518eae-3a75-40d5-861c-b108227aeb0a




## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
